src/SubmitPage/SubmitForm.js

```diff
@@ -249,6 +249,7 @@ class SubmitForm extends React.Component {
                 onChange={e => this.onChangeInput(e, STORE_ADDRESS)}
                 value={formData[STORE_ADDRESS]}
                 placeholder={translations.store_address_placeholder}
+                required
```

Comment:
Is this intended ?

Reply:
@arjun27 is this a required field on backend too ?

Reply:
Address is not required anymore (after #202)